pkg/cli/kubernetes/kubernetes.go

```diff
@@ -172,3 +201,15 @@ type Impl struct {
 func (i *Impl) GetKubeContext() (*api.Config, error) {
 	return kubeutil.LoadConfigFile("")
 }
+
+func (i *Impl) DeleteNamespace(kubeContext string) error {
+	output.LogInfo("Deleting namespace: %s", helm.RadiusSystemNamespace)
```

Comment:
Can you move this logging into the command itself? We try to avoid logging in generic/reusable codepaths in the CLI.